### ISSUE TITLE
Suggestion: schematic files to cards, and vice versa!

### ISSUE THREAD

**Issue by winsrp** [McJty/RFTools#1715]
Since we now have the scanner, which makes shape cards a really powerful creation tool along the builder, could it be possible to have a way to load schematics into shape cards? and vice versa? Schematics are a really popular way for minecrafters to share creations, and having the builder make those in a survival map with just materials would be beyond awesome!

https://minecraft.gamepedia.com/Schematic_file_format

It would be also really cool to have a schematic pack with some of the buildings, roads and features, from your lost city mod, so we could just pick and load into a shape card, and build a city from materials!!!

I already imagined it, and looks awesome, we could even combine the city schematics into a single card and project it, to see how the landscape would be terraformed even before starting, like a live city building preview! HOW AWESOME WOULD THAT BE!!!

Thanks

-Winsrp

PD: YOU ARE THE BEST MINECRAFT MODDER OUT THERE!

**Comment by McJty** [comment]
You are aware that shape cards can already be saved and loaded to disk in its custom format?

**Comment by winsrp** [comment]
yes, but then don't load the billion of schematics already available to download. =D
